index.php

```diff
@@ -9,50 +9,20 @@
     $s = $_GET["s"];
   }
   $title = "CMU Buggy Alumni Association";
-  $headline = "Records: 2:02.16, 2:23.27";
```

Comment:
Appears to be unused everywhere

Reply:
Yeah, I just noticed this.  If so, we should remove all of them, not just some (e.g. history case further down in index.php, as well as the uses in gallery/themes/cmubuggy/views/page.html.php and news/wp-content/themnese/cmubuggy-child/header.php)